pkgs/tools/graphics/waifu2x-ncnn-vulkan/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, vulkan-headers
+, vulkan-loader
+, glslang
+, libgcc
+, libwebp
+, ncnn
+}:
+
+stdenv.mkDerivation rec {
+  pname = "waifu2x-ncnn-vulkan";
+  version = "20220419";
+
+  src = fetchFromGitHub {
+    owner = "nihui";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-cyPqbVEbASZdbUTHnlC8zDmDMiziwbEL9WDcCdN4GFo=";
+  };
+  sourceRoot = "source/src";
+
+  patches = [
+    ./cmakelists.patch
+    ./models_path.patch
+  ];
+
+  cmakeFlags = [
+    "-DUSE_SYSTEM_NCNN=1"
+    "-DUSE_SYSTEM_WEBP=1"
+    "-DGLSLANG_TARGET_DIR=${glslang}/lib/cmake"
+  ];
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ vulkan-headers vulkan-loader glslang libwebp ncnn ]
+    ++ lib.optional (!stdenv.isDarwin) libgcc;
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share
+    install -Dm555 waifu2x-ncnn-vulkan $out/bin/
+    cp -r ${src}/models $out/share
+  '';
```

Comment:
```suggestion
    runHook postInstall
  '';
```

Reply:
Done, also added that for `realesrgan-ncnn-vulkan`.